### DISCUSSION TITLE
Support for 40x2 Character LCDs ala TX802 & TQ5

### DISCUSSION THREAD

**Discussion by now-its-dark** [probonopd/MiniDexed#251] [Ideas]
Hi there! This is something that has crossed my mind since becoming aware of the project, wondering how complex it would be to support higher character-count LCDs, particularly 40x2, as seen in the TX802 and TQ5. This extra space allows for each voice to have a status displayed at once, such as pan states or midi channel assignments.

As a user of the TQ5, I've always appreciated how this speeds up configuring a bank of voices— paired with 8 visually corresponding buttons, it makes for a very fast and direct interaction with the device. 

It it were possible, these [can be found for cheap](https://www.aliexpress.com/item/1704895535.html), so the choice would be more about size and portability of the build than cost.
![tq5MultiVoiceSelect](https://user-images.githubusercontent.com/5728860/169701257-d12a5ba9-37bf-48cf-af11-01b3595dc3a3.gif)

**Comment by probonopd** [comment]
Hello @now-its-dark. "Cheap" is relative, those displays are ~10x as expensive (shipped) as the ones that MiniDexed is currently using. And they won't fit into typical Raspberry Pi sized housings. 

It is probably more realistic to target 128*64 pixel bitmap SSD1306 based OLED displays. They are small, inexpensive, and can show graphics (e.g., great for showing the algorithm).

![image](https://user-images.githubusercontent.com/2480569/169708935-950e42a7-a848-4fb8-951c-facdff5500c6.png)
